pyne/source_sampling.pyx

```diff
@@ -427,6 +481,9 @@ cdef class Sampler:
         if types <= self._sampler_sampler_1_argtypes:
             self._sampler_sampler_1(*args, **kwargs)
             return
+        if types <= self._sampler_sampler_2_argtypes:
```

Comment:
Shouldn't this be an `elif`?

Reply:
Is this a style question or a functional question?  The `return` in the previous block makes this effectively and `elif` anyway...

Reply:
I suppose it is a style issue.  If something is logically an elif, it seems like it should be represented as an elif.  There is also probably some tribal performance difference since using an elif is one if-block where as here it is two if-blocks. This is non-blocking to me, just recommended.